models/migrations/migrations.go

```diff
@@ -406,6 +406,9 @@ var migrations = []Migration{
 	NewMigration("Drop old CredentialID column", dropOldCredentialIDColumn),
 	// v223 -> v224
 	NewMigration("Rename CredentialIDBytes column to CredentialID", renameCredentialIDBytes),
+
+	// v999
```

Comment:
Why is this v999?

Reply:
in case any migrations were merged before and would prevent conflicts, will be changed to correct version before it gets merged.

Reply:
The `v999.go` goes into main branch .............

Reply:
> The `v999.go` goes into main branch .............

Don't worry, just filename is wrong. Database is still right.

Reply:
Will be fixed in #20843

Reply:
I have no worry about it. But I think such development workflow is very unfriendly for developers who need to write migrations (IIRC I have asked about the problem in discord before).

Reply:
> I have no worry about it. But I think such development workflow is very unfriendly for developers who need to write migrations (IIRC I have asked about the problem in discord before).

Hm, yes. About the development workflow, if you have good idea, please share with us and I think we can discuss it.